main/src/library/Process.flix

```diff
@@ -149,6 +210,33 @@ mod Process {
                     case ex: IOException                   => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
 
+            def stderr(ph, k) =
+                try {
+                    let ProcessHandle(jProc) = ph;
+                    k(StdErr.StdErr(jProc.getErrorStream()))
+                } catch {
+                    case ex: IOException => Err(IoError(ErrorKind.Other, ex.getMessage()))
+                }
+
+            def stdin(ph, k) =
+                try {
+                    let ProcessHandle(jProc) = ph;
+                    k(StdIn.StdIn(jProc.getOutputStream()))
+                } catch {
+                    case ex: IOException => Err(IoError(ErrorKind.Other, ex.getMessage()))
+                }
+
+            def stdout(ph, k) =
+                try {
+                    let ProcessHandle(jProc) = ph;
+                    k(StdOut.StdOut(jProc.getInputStream()))
+                } catch {
+                    case ex: IOException => Err(IoError(ErrorKind.Other, ex.getMessage()))
```

Comment:
Can this exception happen here? When calling getInputStream? Or is it not only when reading from the stream?

Reply:
When will it be null?